src/explore-education-statistics-admin/src/pages/release/data/ReleaseAncillaryFilePage.tsx

```diff
@@ -42,15 +42,18 @@ export default function ReleaseAncillaryFilePage({
 
   const handleSubmit = useCallback(
     async ({ title, summary, file: newFile }: AncillaryFileFormValues) => {
-      await Promise.all([
-        releaseAncillaryFileService.updateFile(releaseId, fileId, {
+      if (newFile) {
+        await releaseAncillaryFileService.replaceFile(releaseId, fileId, {
           title,
           summary,
-        }),
-        newFile
-          ? releaseAncillaryFileService.replaceFile(releaseId, fileId, newFile)
-          : undefined,
-      ]);
+          file: newFile,
+        });
+      } else {
+        await releaseAncillaryFileService.updateFile(releaseId, fileId, {
+          title,
+          summary,
+        });
+      }
```

Comment:
If we fix the semantics from 'replace' to 'update', we can just call a single endpoint for updating the ancillary file.